{"uid":"f56da9570cfa5f6c","name":"test_list1","fullName":"unit_tests.test_add.TestAdd#test_list1","historyId":"6f9ec05af584e70b86f4005288e423e7","time":{"start":1749841919443,"stop":1749841919444,"duration":1},"status":"skipped","statusMessage":"XFAIL\n\nAssertionError: assert [1, 2, 3, 4] == [1, 2, 3, 4, 5]\n  \n  Right contains one more item: 5\n  \n  Full diff:\n    [\n        1,\n        2,...\n  \n  ...Full output truncated (4 lines hidden), use '-vv' to show","statusTrace":"self = <test_add.TestAdd object at 0x00000299DC1E10D0>\n\n    @pytest.mark.xfail  # expected to be failed, and actual failed, xfailed\n    @pytest.mark.api\n    def test_list1(self):\n        res = add([1], [2, 3, 4])\n>       assert res == [1, 2, 3, 4, 5]\nE       AssertionError: assert [1, 2, 3, 4] == [1, 2, 3, 4, 5]\nE         \nE         Right contains one more item: 5\nE         \nE         Full diff:\nE           [\nE               1,\nE               2,...\nE         \nE         ...Full output truncated (4 lines hidden), use '-vv' to show\n\nunit_tests\\test_add.py:53: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"ff","time":{"start":1749841917877,"stop":1749841917877,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"stepsCount":0,"attachmentStep":false,"attachmentsCount":0},{"name":"f","time":{"start":1749841917877,"stop":1749841917877,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"stepsCount":0,"attachmentStep":false,"attachmentsCount":0}],"testStage":{"status":"skipped","statusMessage":"XFAIL\n\nAssertionError: assert [1, 2, 3, 4] == [1, 2, 3, 4, 5]\n  \n  Right contains one more item: 5\n  \n  Full diff:\n    [\n        1,\n        2,...\n  \n  ...Full output truncated (4 lines hidden), use '-vv' to show","statusTrace":"self = <test_add.TestAdd object at 0x00000299DC1E10D0>\n\n    @pytest.mark.xfail  # expected to be failed, and actual failed, xfailed\n    @pytest.mark.api\n    def test_list1(self):\n        res = add([1], [2, 3, 4])\n>       assert res == [1, 2, 3, 4, 5]\nE       AssertionError: assert [1, 2, 3, 4] == [1, 2, 3, 4, 5]\nE         \nE         Right contains one more item: 5\nE         \nE         Full diff:\nE           [\nE               1,\nE               2,...\nE         \nE         ...Full output truncated (4 lines hidden), use '-vv' to show\n\nunit_tests\\test_add.py:53: AssertionError","steps":[],"attachments":[{"uid":"3396388fc19a4f3c","name":"log","source":"3396388fc19a4f3c.txt","type":"text/plain","size":243}],"parameters":[],"hasContent":true,"shouldDisplayMessage":true,"stepsCount":0,"attachmentStep":false,"attachmentsCount":1},"afterStages":[{"name":"f::0","time":{"start":1749841919566,"stop":1749841919566,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"stepsCount":0,"attachmentStep":false,"attachmentsCount":0}],"labels":[{"name":"tag","value":"api"},{"name":"parentSuite","value":"unit_tests"},{"name":"suite","value":"test_add"},{"name":"subSuite","value":"TestAdd"},{"name":"host","value":"Novele"},{"name":"thread","value":"26108-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"unit_tests.test_add"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["api"]},"source":"f56da9570cfa5f6c.json","parameterValues":[]}